snap/info.go

```diff
@@ -459,6 +493,12 @@ func (s *Info) InstanceName() string {
 	return InstanceName(s.SnapName(), s.InstanceKey)
 }
 
+// ContName returns the name of the container, which the instance name for
+// snaps.
+func (s *Info) ContName() string {
```

Comment:
This should be the full name, usually `cont` is short for `continue`
```suggestion
func (s *Info) ContainerName() string {
```

Reply:
Changed